orchestrators/ProcurementOperations/src/Services/PaymentProcessingService.php

```
@@ -113,622 +77,175 @@ public function createBatch(
                 vendorName: $vendorName,
-                paymentAmount: $paymentAmount,
-                payeeName: $vendorName,
-                mailingAddress: 'Address on file',
-                discountAmount: $discountAmount,
-                remittanceInfo: $remittanceInfo,
+                amount: $amount,
+                invoiceIds: $invoiceIds,
+                paymentReference: 'MULTIPLE',
+                checkNumber: $vendorBankAccount,
             ),
             default => throw new \InvalidArgumentException(
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '124', column '34'). (undefined)

(MissingImport)